services/app-web/src/common-code/proto/healthPlanFormDataProto/toDomain.ts

```diff
@@ -29,7 +29,7 @@ type RecursivelyReplaceNullWithUndefined<T> = T extends null
               : RecursivelyReplaceNullWithUndefined<T[K]>
       }
 
-export function replaceNullsWithUndefineds<T>(
+export function replaceNullsWithUndefineds<T extends object>(
     obj: T
 ): RecursivelyReplaceNullWithUndefined<T> {
```

Comment:
@macrael @haworku @rswerve @mojotalantikite The generic was causing ts errors and constraining it the generic to `object` fixed this. Could someone take a look at this and approve/merge this PR if the changes I made looks good?